tool/src/webapp/templates/ShowPage.html

```diff
@@ -51,8 +51,11 @@
 <link rel="stylesheet" href="$context/js/jquery.oembed.css" type="text/css" />
 <script type="text/javascript" src="$context/js/jquery.cluetip.min.js"></script>
 <link rel="stylesheet" href="$context/css/jquery.cluetip.css" type="text/css" />
+<link rel="stylesheet" href="../../library/editor/ckextraplugins/common-wl/css/file-tree.css" type="text/css"/>
```

Comment:
You shouldn't have this as a relative reference. `/library` references should alway start from the root (`/`).